app/main.py

```diff
@@ -2,0 +1,37 @@
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self._hidden: bool = False
+        Animal.alive.append(self)
+
+    @property
+    def hidden(self) -> bool:
+        return self._hidden
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, Health: {self.health}, "
+                f"Hidden: {self._hidden}}}")
```

Comment:
Code Style: Use double quotes everywhere. You should use double quotes instead of single quotes in your string literals. This is more consistent with the rest of your code, and it's also the recommended style in PEP 8, the Python style guide.